apps/cloudo/src/events/authorized.ts

```diff
@@ -0,0 +1,21 @@
+import {
+	ApplicationIntegrationType,
+	ApplicationWebhookEventType,
+	type Client,
+	Listener,
+	type ListenerEventData
+} from "@buape/carbon"
+
+export default class ApplicationAuthorizedListener extends Listener {
+	type = ApplicationWebhookEventType.ApplicationAuthorized
+	override async handle(
```

Comment:
Because tsconfig noImplicitOverride is not enabled the override keyword isn't required
```suggestion
	async handle(
```